src/sparseml/pytorch/optim/manager.py

```diff
@@ -32,7 +31,185 @@
 from sparsezoo.objects import Recipe
 
 
-__all__ = ["ScheduledModifierManager", "load_manager"]
+__all__ = ["ModifierManagerOptimWrapper", "ScheduledModifierManager"]
+
+
+class ModifierManagerOptimWrapper(object):
```

Comment:
nit: not sure if this name makes it clear what the class' purpose is. could we change the name to something like `TrainingStepRecipeWrapper` to singal that we're wrapping things that aren't just solo optimizers?

Reply:
Will rename to RecipeManagerStepWrapper in prep for renaming the Managers to RecipeManager for phase2